.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -2,10 +2,9 @@
 
 <!--
   To start the checklist for a new release:
-  - If it's an internal release, open a new issue in dotnet/core-eng (private repo).
+  - If it's an internal release, open a new issue in dotnet/release (private repo).
     - Otherwise, open a new issue in dotnet/source-build.
-  - Paste one copy of the below checklist per release. (Once for 2.1, again for 3.1, if both are being serviced)
-  - Delete lines that only apply to other releases. (E.g. delete [2.1] if doing a 3.1 release.)
+  - Paste a copy of the below checklist.
   - Delete lines starting with [Internal] if running a non-internal release.
   - Delete lines starting with [Non-Internal] if running an internal release.
```

Comment:
Are we going to have non-internal releases for 3.1?  If there are, to simplify this workflow, what would be wrong with treating all released as internal?  It would reduce the chances of errors.